stable/sonarqube/values.yaml

```diff
@@ -212,3 +212,5 @@ mysql:
 # podLabels:
 #   key: value
 podLabels: {}
+# For compatibility with 8.0 replace by "sq"
+runShFolder: sonarqube
```

Comment:
stable/sonarqube/values.yaml
  216:23    error    no new line character at the end of file  (new-line-at-end-of-file)